src/quo2/components/wallet/account_card/component_spec.cljs

```diff
@@ -6,7 +6,7 @@
 (def empty-username "Account 1")
 
 (defn get-test-data
-  [type watch-only? empty-type? loading? metrics?]
+  [{:keys [type watch-only? empty-type? loading? metrics?] :or {metrics? true}}]
```

Comment:
given that this is only a component test it's not such a big issue, however imo when creating an api it is better to default the value to false. 
consider changing the name to the negation and remove the default value. e.g:  use`:no-metrics?` instead.
it's strange to have to set `false` to a prop value.

Reply:
Done